packages/@azure/servicebus/data-plane/test/stress/singleClient.ts

```diff
@@ -0,0 +1,61 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+const queueName = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
+  const ns = ServiceBusClient.createFromConnectionString(connectionString);
+
+  try {
+    while (!isJobDone) {
+      const client = ns.createQueueClient(queueName);
+      const sender = client.createSender();
+
+      const message: SendableMessageInfo = {
+        messageId: msgId,
+        body: "test",
+        label: `${msgId}`
+      };
+      msgId++;
```

Comment:
This is not checked when we receive the message.

Reply:
Since we already did this in single queue case, this is covered as the message count would be specific to queue and client. (Which is same as what we do in singleMessageComplete test).
Hence, I've not added in verification of message. (Also, because I thought the focus in this test was on client creation.)
Will defer it to you if we want to still add in the message verification.

Reply:
If we don't do the verification should we remove the code around the `msgId`?

Reply:
For troubleshooting any issues with tests should they occur, it's useful to have it in.

Reply:
All right.